src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
You must define it like `Animal.alive` just like in readme file

Reply:
you still reference to it like Animal.alive